### PR TITLE
New recoil corrections and batch job arrays

### PR DESCRIPTION
**Pull request by 12345ieee**

### CODE REVIEW

File: CMGTools/WMass/analysis/includes/common.h [hunk 1]
```diff
@@ -55,7 +55,7 @@ namespace WMass{
   // static const int RecoilCorrVarDiagoParN_[]                  = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11 };
   
   static const int RecoilCorrIniVarDiagoParU1orU2fromDATAorMC_[] = {0, 0,  9,  0, 0,  9,  0 };
-  static const int RecoilCorrNVarDiagoParU1orU2fromDATAorMC_[]   = {1, 9, 18, 12, 9, 18, 12 };
+  static const int RecoilCorrNVarDiagoParU1orU2fromDATAorMC_[]   = {1, 9, 21, 15, 9, 21, 15 };
```
Comment (perrozzi) [inline CMGTools/WMass/analysis/includes/common.h:58]: sono aumentati i parametri perche' non ci sono piu' i constraint sulle medie?

**Comment by perrozzi** [comment]
l'unica cosa che mi devo guardare meglio un secondo e' l'array di bsub, che non ho mai usato quindi poi mi vedo un po' che fa esattamente, per il resto direi tutto ok, faccio merge. grazie!

File: CMGTools/WMass/analysis/includes/common.h (hunk 1 above)
Comment (12345ieee) [inline CMGTools/WMass/analysis/includes/common.h:58]: Correct

**Comment by 12345ieee** [comment]
Tutto quello che so sugli array l'ho imparato da qui: http://www.ccs.miami.edu/hpc/lsf/7.0.6/admin/jobarrays.html

**Comment by 12345ieee** [comment]
To add on the jobs arrays, the first tests were very successful, the PEND time was roughly the same, but the submission was much, much faster.

These last days, though, my jobs stay on PEND state for hours and hours.
I don't know if it's due to the usage of job arrays, or something else, but it's something to keep in mind.

**Comment by perrozzi** [comment]
ciao, apparently there is a lxbatch problem.
if you do bqueues there are tens of thousands of jobs pending on several
queues... :-(

**Pull request merged**